src/index.html

```diff
@@ -1,14 +1,18 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="watch">
+      <div class="watch__second-hand watch__second-hand--style"></div>
+      <div class="watch__minute-hand watch__minute-hand--style"></div>
+      <div class="watch__center watch__center--style"></div>
+    </div>
```

Comment:
Improve the BEM structure. You can do it like this
```suggestion
    <div class="watch">
      <div class="watch__hand watch__hand--second"></div>
      <div class="watch__hand watch__hand--minute"></div>
      <div class="watch__center"></div>
    </div>
```

Reply:
ok, thanks)
 so much easier)